ui/ducks/send/send.js

```diff
@@ -571,6 +571,8 @@ export const computeEstimatedGasLimit = createAsyncThunk(
  * the send slice. It returns the values that might change from this action and
  * those values are written to the slice in the `initializeSendState.fulfilled`
  * action handler.
+ *
```

Comment:
NBD two comments that reference `actions.js` instead of `actions.ts`. Commenting here because 1 instance is in this file

![Screen Shot 2023-02-04 at 12 32 42 AM](https://user-images.githubusercontent.com/20778143/216672260-77d0bac7-b59b-4427-8f2c-c02b069ec7cb.png)

Reply:
Thanks for pointing that out. The files-to-convert does need to be .js cause the dashboard will turn the box green for this file cause its now .ts :D